src/solver/lbfgs.jl

```diff
@@ -3,9 +3,14 @@ export lbfgs
 function lbfgs(nlp :: AbstractNLPModel;
                atol :: Float64=1.0e-8, rtol :: Float64=1.0e-6,
                max_f :: Int=0,
+               max_iter :: Int=typemax(Int),
```

Comment:
I don't think iterations are meaningful. It's too ambiguous to compare one iteration of a solver to that of another.